compiler/parser.nim

```diff
@@ -1254,12 +1250,14 @@ proc parseObject(p: var Parser): PNode
 proc parseTypeClass(p: var Parser): PNode
 
 proc primary(p: var Parser, mode: PrimaryMode): PNode =
-  #| typeKeyw = 'var' | 'out' | 'ref' | 'ptr' | 'shared' | 'tuple'
-  #|          | 'proc' | 'iterator' | 'distinct' | 'object' | 'enum'
-  #| primary = typeKeyw optInd typeDesc
+  #| primary = operatorB primary primarySuffix* |
+  #|           tupleDecl | routineExpr | enumDecl
+  #|           objectDecl | conceptDecl | ('bind' primary)
+  #|           ('var' | 'out' | 'ref' | 'ptr' | 'distinct') primary
   #|         /  prefixOperator* identOrLiteral primarySuffix*
-  #|         / 'bind' primary
   if isOperator(p.tok):
+    # Note 'sigil like' operators are currently not reflected in the grammar
+    # and should be removed for Nim 2.0, I don't think anybody uses them.
```

Comment:
`var someSeq = @[1, 2, 3]` ?

Reply:
That's just `@` used as an ordinary unary operator.

Reply:
Do you think nobody uses `@[1, 2, 3].foo()`? I think it's used a lot, and changing precedence of an operator is one of the worst unnoticable breaking changes possible.

Reply:
As far as I understand the situation that code too would continue to work. Maybe I'm wrong and anyway it's just a comment, not an RFC process...